sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -167,8 +168,7 @@ def _generate_sas_token(uri, policy, key, expiry=None):
     token = generate_sas_token(uri, policy, key, abs_expiry)
     return AccessToken(token=token, expires_on=abs_expiry)
 
-def _build_uri(address, entity):
-    # type: (str, Optional[str]) -> str
+def _build_uri(address: str, entity: Optional[str]) -> str:
```

Comment:
I think we should just make this `str`. Passing in `None` will raise an exception, so why allow it

```suggestion
def _build_uri(address: str, entity: str) -> str:
```